pkg/resources/controllermanager/deployment.go

```diff
@@ -273,7 +273,7 @@ func getFlags(data *resources.TemplateData, version *semverlib.Version) ([]strin
 	if cloudProviderName != "" && cloudProviderName != "external" {
 		flags = append(flags, "--cloud-provider", cloudProviderName)
 		flags = append(flags, "--cloud-config", "/etc/kubernetes/cloud/config")
-		if cloudProviderName == "azure" && version.Minor() >= 15 {
+		if cloudProviderName == "azure" && version.GreaterThan(semverlib.MustParse("1.15.0")) {
```

Comment:
It is actually not, good catch. I guess it doesn't make a difference because we're long past 1.15 as a supported version, but let me fix this real quick.

Reply:
fixed with a constraint.